rust/core-lib/src/rpc.rs

```diff
@@ -288,7 +288,11 @@ pub struct EditCommand<T> {
 #[derive(Serialize, Deserialize, PartialEq, Eq, Debug)]
 #[serde(rename_all = "snake_case")]
 pub enum GestureType {
+    Click,
```

Comment:
Maybe `Click` could be `SetSelection`? Or `Select` or `PointSelect`? Not sure, open to other names. My thinking is that one nice thing about moving to this gesture model is that it lets us abstract away the particulars of the hardware; users on touch devices don't 'click', but they do still send whatever this gesture is.

Reply:
I agree. I couldn't come up with a better name at the time but I like `PointSelect`.